sdk/src/main/java/com/horizen/account/api/rpc/types/EthereumBlockView.java

```diff
@@ -0,0 +1,69 @@
+package com.horizen.account.api.rpc.types;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import com.horizen.account.block.AccountBlock;
+import com.horizen.serialization.Views;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@JsonView(Views.Default.class)
+public class EthereumBlockView {
+    public final Long number;
+    public final String hash;
+    public final String parentHash;
+    public final String nonce;
+    public final String sha3Uncles;
+    public final String logsBloom;
+    public final String transactionsRoot;
+    public final String stateRoot;
+    public final String receiptsRoot;
+    public final String miner;
+    public final String mixHash;
+    public final String extraData;
+    public final String size;
+    public final String gasLimit;
+    public final String gasUsed;
+    public final String timestamp;
+    public final List<?> transactions;
+    public String author;
+    public String difficulty;
+    public String totalDifficulty;
```

Comment:
still not used. check the comments in the task

Reply:
@GitMark0 I see that these two fields has been removed at all now.
It is the issue.
Instead we need to:
1) keep this two fields
2) assign `difficulty = 1` always
3) assign `totalDifficulty = blockNumber`

Reply:
Shouldn't the difficulty be zero? In Ethereum PoS it is always `difficulty = 0x0` imho. I don't know what `totalDifficulty` is in Ethereum now though.

Example response by Alchemy RPC on current Ethereum Mainnet:
```
{
  "jsonrpc": "2.0",
  "id": 0,
  "result": {
    "baseFeePerGas": "0x4696470c1",
    "difficulty": "0x0",
    "extraData": "0x496c6c756d696e61746520446d6f63726174697a6520447374726962757465",
    "gasLimit": "0x1c9c380",
    "gasUsed": "0xb68af9",
    "hash": "0x3cb935908566e5f255c08629838aa10107a1e18c9c975ad8567ebf6142389339",
    "logsBloom": "0x1c212572e1c3eb4213942454d130922af92bc800719ac20eab99c0255626981240022fb68d51400b2334b84a520f29f132731ed45fc8b84fc2c16b00d02c296961834d005c6f01296b4b2008902007a5929712a048448edd01d21de08601203a126102c1323e811a10c830853b6298cd0c6c0c63c735366c620441522dfd48621374077c906a104984df9c090938110084013945815084b9507650c439701a03bf73514f40007011800574e89c4e366c4b41862f988ea882682626c885e909156540c6f2b98d5af01c1289132807360683b73ed08aaaa050dd9c9fc6dc41f45440b5e948a40352146083318908411c8aa50818500b6807f4682f0e43d8127113",
    "miner": "0xdafea492d9c6733ae3d56b7ed1adb60692c98bc5",
    "mixHash": "0xe8a098ab9940c376e1d29c02d803b1b7e4d094728964ce2596955b4a4cf48824",
    "nonce": "0x0000000000000000",
    "number": "0xf612bf",
    "parentHash": "0xd8dc01fefab792f2816f113845fb3a45cc46157efdeb9a334ef60d0e7ec6df77",
    "receiptsRoot": "0x80b415f75be82fa9742257a9606dcee7541b123ff2d1ce2f25caf227c2655ccf",
    "sha3Uncles": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
    "size": "0x15689",
    "stateRoot": "0x0bcf4463b56e8fc2a5d2ce7044b874b55a778cebed88b1b542e3c2bee587204a",
    "timestamp": "0x638f5fdf",
    "totalDifficulty": "0xc70d815d562d3cfa955",
    "transactions": [
      "0xb33042a944ebaa36a692e60220693b364e9fc9b54be5c716ff7c8a76ad4a8c2b",
      "..... removed"
    ],
    "transactionsRoot": "0x402ccf0682a11b7cb5cb5a108faeaa202fce71ca2b271dbc4606d9fb3d35a1e7",
    "uncles": []
  }
}
```

Reply:
Just confirming that the `difficulty` param should be `0` since totalDifficulty is calculated as the total of the difficulty of all the previous blocks plus the previous blocks. We can see, from Etherscan, that the `totalDifficulty` didn't change since the merge.

Reply:
There are some weird things though, they said that difficulty bomb hits at: `58,700,000,000,000,000,000,000`, but current `totalDifficulty` is `58,750,003,716,598,352,816,469` and that number didn't change in last 10 blocks that I've checked...

Reply:
The difficulty bomb was delay multiple times, the final terminal total difficulty was set to `58_750_000_000_000_000_000_000`.

If the `totalDifficulty` should be the sum of the `difficulty` of all previous blocks then I assume, for us, a zero is correct here too. We need to output `difficulty` and `totalDifficulty` just for compatibility, they don't have a meaning for us - just like it is on Ethereum mainnet now, after the merge.